soccer/SystemState.cpp

```diff
@@ -20,38 +21,41 @@ class BallPath : public Planning::Path {
 public:
     BallPath(const Ball& ball) : ball(ball){};
 
-    virtual bool hit(const Geometry2d::ShapeSet& obstacles,
-                     RJ::Seconds startTimeIntoPath,
-                     RJ::Seconds* hitTime) const {
-        throw new std::runtime_error("Unsupported Opperation");
+    bool hit(const Geometry2d::ShapeSet& /*obstacles*/,
+             RJ::Seconds /*startTimeIntoPath*/,
+             RJ::Seconds* /*hitTime*/) const override {
```

Comment:
I assume this is to stop any unused variable warnings or something?

Reply:
Yeah. `misc-unused-parameters` [link](https://clang.llvm.org/extra/clang-tidy/checks/misc-unused-parameters.html)